### DISCUSSION TITLE
Nivo radar theme object

### DISCUSSION THREAD

**Discussion by Streammer** [plouc/nivo#1752] [Q&A]
Hello, I have to customize nivo radar. Where could I find documentation about all features in theme object. In that link https://nivo.rocks/guides/theming I had found only a few general features. Also I need to set a custom colors, not from sheme.

**Comment by wyze** [comment]
What color are you trying to modify? There are several available: https://nivo.rocks/radar/ .. dotColor, dotBorderColor, borderColor.
